api/test/integration/env/configurations/vulnerability/manager/healthcheck/healthcheck.py

```diff
@@ -30,8 +30,8 @@ def get_agents_node(status='active'):
         Dictionary like {agent_id: manager_name} if agents were obtained, else None.
     """
     try:
-        headers['Authorization'] = f'Bearer {get_response(login_url, headers)["data"]["token"]}'
-        response = get_response(base_url + f'/agents?select=id,manager&status={status}',
+        headers['Authorization'] = f'Bearer {get_response("POST", LOGIN_URL, headers)["data"]["token"]}'
+        response = get_response('GET', f'{BASE_URL}/agents?select=id,manager&status={status}',
```

Comment:
Be consistent with quotation, please. Also, remember to use double quotation marks on f-strings.